migrate/v0_15/migrate.go

```diff
@@ -174,11 +175,38 @@ func Committee(genesisState v0_14committee.GenesisState) v0_15committee.GenesisS
 			baseStabilityCom := stabilityCom.SetPermissions(newStabilityCommitteePermissions)
 			newStabilityCom := v0_15committee.MemberCommittee{BaseCommittee: baseStabilityCom}
 			committees = append(committees, newStabilityCom)
-		} else {
+		case 2:
 			safetyCom := comtypes.NewMemberCommittee(com.ID, com.Description, com.Members,
 				[]v0_15committee.Permission{v0_15committee.SoftwareUpgradePermission{}},
 				com.VoteThreshold, com.ProposalDuration, v0_15committee.FirstPastThePost)
 			committees = append(committees, safetyCom)
+		case 3:
```

Comment:
I think we need to create the swap committee outside the switch, then append it, since there will be no incoming committee with id=3.

Reply:
https://github.com/Kava-Labs/kava/pull/940/commits/084a00790d38befdadf211fea012df88c0d6f6df